build_conf.py

```diff
@@ -165,6 +168,9 @@ def __parse_args(self):
 		parser.add_argument('--with-sdk', action='store_true',
 							dest='populate_sdk', required=False, default=False,
 							help='Populate SDK for the build')
+		parser.add_argument('--with-sstate-cache', action='store_true',
```

Comment:
We always build with sstate, so I would suggest renaming the parameter to have populate word in it